app/models/location.rb

```diff
@@ -0,0 +1,11 @@
+class Location < ActiveRecord::Base
+  validates :name, presence: true
+  validates :name, uniqueness: { case_sensitive: false }
+  validates :state, inclusion: { in: ["AL",
```

Comment:
Ignoring the Hound errors below, consider putting this array into a constant: 

```
class Location
STATES = [...]

validates :state, inclusion: { in: STATES }
```

This doesn't prevent a merge IMO, just thoughts for the future.